aicsimageio/aics_image.py

```diff
@@ -309,6 +309,38 @@ def metadata(self) -> Any:
         # The reader can implement read optimization or not.
         return self.reader.metadata
 
+    def get_ome_metadata(self):
+        """
+        Get the OME transformed metadata by applying the XSLT to the native
+        metadata format.
+
+        Returns
+        -------
+        lxml.etree.Element
+            The metadata converted into OME format using the XSLT sheets.
+        """
+        return self.reader.get_ome_metadata()
+
+    def get_single_scene_ome_metadata(self, scene_index: int = 0):
```

Comment:
Why can't this be done outside of here?  It actually looks like something that should be a member function in OMEXML.py.

Reply:
Yeah, second this. I think this should be either a utility function in it's own file.

I was thinking like:
`aicsimageio/metadata_utils.py`

```python
def select_single_scene_from_ome_metadata(ome_xml: ET.Element, scene_index: int = 0):
    ...
```

So you pass in the returned XML from the XSLT and select down. More reusable than running the transform multiple times.